packages/adapter-fauna/README.md

```diff
@@ -0,0 +1,29 @@
+# `@lucia-auth/adapter-supabase`
```

Comment:
Make sure to change `supabase` to `Fauna`!

Reply:
I guess I was so happy my adapter worked I overlooked the readme 😅 sorry